basiclti/basiclti-blis/src/java/org/sakaiproject/lti13/LTI13Servlet.java

```diff
@@ -295,6 +301,60 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
 
 	}
 
+	// A very locked down proxy - JSON Only
+	protected void handleProxy(HttpServletRequest request, HttpServletResponse response) {
+		String proxyUrl = request.getParameter("proxyUrl");
+		if ( proxyUrl == null ) {
+			LTI13Util.return400(response, "Missing proxyUrl");
+			return;
+		}
+
+		Session sess = SessionManager.getCurrentSession();
+		if ( sess == null || sess.getUserId() == null ) {
+			LTI13Util.return400(response, "Must be logged in");
+			return;
+		}
+
+		try {
+			java.net.URL url = new java.net.URL(proxyUrl);
+			java.net.HttpURLConnection con = (java.net.HttpURLConnection) url.openConnection();
+			con.setRequestMethod("GET");
+			con.setConnectTimeout(3000);
+			con.setReadTimeout(3000);
+			con.setInstanceFollowRedirects(true);
```

Comment:
these 4 lines seem like defaults ?  why explicitly set them ?

Reply:
Really?  Let me check.

Reply:
I like explicitly listing these even if they are the defaults - to be really obvious about what is going on for the casual reader.